suppressions/ruby.supp

```diff
@@ -34,3 +34,19 @@
   fun:each_location*
   ...
 }
+{
+  Rust probes for statx(buf)
```

Comment:
We definitely could, but it does make the barrier to entry higher to use this gem. IMO, the trade-off doesn't seem worth it since adding these two suppressions doesn't really come with much cost AFAICT. But I'll leave the final decision up to you.

Reply:
It looks like it was released as part of Valgrind 3.16.0. Ubuntu 22.04 repositories have Valgrind 3.15.0. I guess we can include these suppressions.

Can you add a comment to these suppressions saying that they're fixed in Valgrind >= 3.1.6.0?